modules/resolve-styles.js

```diff
@@ -161,6 +161,20 @@ var resolveStyles = function (
     }
   }
 
+  // Bail early if element is not a simple ReactDOMElement.
+  if (
+    !React.isValidElement(renderedElement) ||
+    typeof renderedElement.type !== 'string'
+  ) {
+    if (oldChildren === newChildren) {
+      return renderedElement;
+    }
+
+    return React.cloneElement(
+      renderedElement, renderedElement.props, newChildren
```

Comment:
if it doesn't fit on one line in 80 chars or so, drop each param to its own line